### PR TITLE
fix CI

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15179/merge/testResults/webgl2playwright/index.html

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/15179/merge/testResults/webgpuplaywright/index.html

File: packages/tools/tests/test/visualization/config.json [hunk 1]
```diff
@@ -2100,11 +2099,6 @@
             "title": "Shadows in RHS mode",
             "playgroundId": "#RO51RT#14",
             "referenceImage": "shadows-rhs.png"
-        },
-        {
-            "title": "Load environment from data buffer",
-            "playgroundId": "#YLI4J1#",
-            "referenceImage": "environment-from-data-buffer.png"
```
Comment (RaananW) [inline packages/tools/tests/test/visualization/config.json:2107]: Did you mean to remove this?

**Review by RaananW — APPROVED** [review]

File: packages/tools/tests/test/visualization/config.json (hunk 1 above)
Comment (deltakosh) [reply packages/tools/tests/test/visualization/config.json:2107]: nope///fixing

**Pull request merged**